errorpage.html

```diff
@@ -1,58 +1,14 @@
-        <title>Error Page</title>
-        <meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<script src="https://use.fontawesome.com/d3acecdb64.js"></script>
-<link
-    href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-    rel="stylesheet">
-<link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;600;700;900&display=swap"
-      rel="stylesheet">
-<link rel="icon" type="image/png" href="imgs/favicon.png">
-<link rel="stylesheet" href="styles/style.css">
-        <link rel="stylesheet" type="text/css" href="errorstyle.css">
-        <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Nunito" />
-        <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
-    </head>
-    <body>
-        <h1>// Goodbye World?</h1>
-        <div id="container">
-            <div id="left"><img src="sadrobot.png" height=400 /></div>
-            <div id="right"><br /><br /><br />The page you're looking for might have been deleted or moved.<br /><br />Sorry!</div>
-        </div><br />
-        <div id="home">RETURN TO HOMEPAGE</div>
-        <p>OR</p>
-        <div id="support">CONTACT SUPPORT</div>
-        <p id="contactinfo" style="font-size:22px"></p>
-        <div id="footer">
-            <div>
-                <a href="https://facebook.com/HelloWorldCodeCamp/">
-                    <span class="fa-stack fa-lg">
-                        <i class="fa fa-circle fa-stack-2x"></i>
-                        <i class="fa fa-facebook fa-stack-1x fa-inverse"></i>
-                    </span>
-                </a>
-                <a href="https://instagram.com/_bitxbit_/">
-                    <span class="fa-stack fa-lg">
-                        <i class="fa fa-circle fa-stack-2x"></i>
-                        <i class="fa fa-instagram fa-stack-1x fa-inverse"></i>
-                    </span>
-                </a>
-                <a href="mailto:info@bitbybitcoding.org">
-                    <span class="fa-stack fa-lg">
-                        <i class="fa fa-circle fa-stack-2x"></i>
-                        <i class="fa fa-envelope fa-stack-1x fa-inverse"></i>
-                    </span>
-                </a>
-            </div>
-            Bit by Bit Coding is a registered nonprofit
-            <br>
-            EIN 84-5025902
-            <br>
-            &copy; 2020 Bit by Bit Coding
-        </div>
-    </body>
-</html>
\ No newline at end of file
+---
+layout: default
+title: "404 Page"
+style: errorstyle.css
+---
+<!--<link rel="stylesheet" type="text/css" href="errorstyle.css">-->
+
+<h1>// Goodbye World?</h1>
+<div id="container">
+    <div id="left"><img src="sadrobot.png" height=400 /></div>
```

Comment:
Can you also upload `sadrobot.png` to the `imgs` folder? currently it's showing up as a blank image